### ISSUE TITLE
[wg/ag] AGWG Charter - Scope & Timeline

### ISSUE THREAD

**Issue by sydneycolemanfromgoogle** [w3c/charter-drafts#790]
Scope and Timeline: Google is requesting a detailed list of what is and is not in scope for WCAG-3 (i.e. a declaration of what changes will be postponed until WCAG 3.1) and a realistic timeline for when the W3C plans to publish WCAG-3. Google would like to see a set of deadlines, defined by the working group, with corresponding areas of focus (similar to a GANTT chart). As part of this, Google is interested in understanding what is in scope for WCAG-EM with regards to sampling and testing and how it relates to statements of conformance.

**Comment by sydneycolemanfromgoogle** [comment]
@jyasskin

**Comment by iadawn** [comment]
A detailed schedule of the work of the group has been developed and maintained for some time. To increase visibility this has been moved into the [WCAG 3 GitHub wiki](https://github.com/w3c/wcag3/wiki#project-management-documents). It is worth noting that the purpose of the charter is not to be a project plan but to provide an overview of the work of the group.

As noted in w3c/charter-drafts#789 the refining the scope of WCAG 3 is a key part of the work as it progresses. One of the requirements of WCAG 3 is to [broaden disability support](https://www.w3.org/TR/wcag-3.0-requirements/#broad-disability-support) and there has been considerable work done to include requirements that address this. As scrutiny of these new requirements progresses it will likely become apparent that some simply cannot be made to work while others will be modified slightly to ensure they can be incorporated. Maturing the requirements over the next two years is on of the key tasks for the group in regards to WCAG 3.

Regarding WCAG-EM specifically, what will be required of this resource will not be apparent until there is more consensus on the approach to conformance for WCAG 3. Again, this is something that is covered as a goal within the new charter.

**Comment by sydneycolemanfromgoogle** [comment]
Again it sounds like you believe that the conformance model should be finalized before proceeding with a policy doc or wcag-em, is that correct?

**Comment by iadawn** [comment]
No.

As you know there are still many questions on what the conformance model will be. The answers to these questions will impact on what is required of resources that are connected to the conformance model. This does not mean that it needs to be 'finalized' before proceeding with any of these resources. However, there is a need for more clarity.

The charter recognizes that there is still work to be done in all of these areas.

**Comment by sydneycolemanfromgoogle** [comment]
Sorry 'finalized' was the wrong word -- widely agreed upon, rather.

**Comment by iadawn** [comment]
My view is that there are already a number of topics within the conformance space that are clearly not within the remit of a standard to address but are questions that policy makers would need to consider. For example, bugs or archival content. So while there is still work to do on the conformance model, there are aspects that will fit into guidance that could be developed. Indeed there is a subgroup being set up to [explore policy guidance](https://www.w3.org/wbs/35422/policy_subgroup_26/?login=).
